configuration/rules-blockly-value-storage.md

```diff
@@ -0,0 +1,61 @@
+---
+layout: documentation
+title: Rules Blockly - Value Storage
+---
+
+# Value Storage
+[return to Blockly Reference]({{base}}/configuration/rules-blockly.html)
+
+## Introduction
+
+These blocks enable storing information *for a rule* that is kept after the rule has run, so it can be reused when the rule is run again later in stateful way. Basically a value storage can be perceived as a global variable for the rule instance.
```

Comment:
Please change this to one sentence per line.
The website build will take core of a proper presentation of paragraphs.